### PR TITLE
Add deployment instructions

### PR DESCRIPTION
**Pull request by arthurevans**
@aomarks Adding deployment instructions per our conversation. Would you take a look?

... I included a CLI instruction for migrating traffic, although maybe it would be simpler (if a little slower) to simply re-run the deploy command with `-- --promote`...

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -54,3 +54,34 @@ API docs unless you're making changes to the doc comments in the lit-element sou
...
+
+```bash
+npm run deploy
+```
+
+(You must have permission to update the App Engine project to deploy the site.)
+
+The site is staged at a new URL (for example, `https://20181207t151022-dot-polymer-lit-element.appspot.com`). When ready, use the App Engine console to migrate traffic to the new version.
+
+To push to a named App Engine version, specify the version on the command line:
```
Comment (aomarks) [inline docs/README.md]: Dumb question, but why would we ever want to push to a specific version? Wouldn't we either stage at the temporary URL shown above, or push to production?

Comment (arthurevans) [reply docs/README.md]: Good question. I do it, for example, when I'm working on a significant new chunk of doc that I expect will take several revisions, and I want people to be able to review it at a stable URL, like:

`https://new-tooling-section-dot-polymer-lit-element.appspot.com`

I considered mentioning that in the PR--I'll go ahead and add that explanation.

Comment (arthurevans) [inline docs/README.md]:
```suggestion
Sometimes it's useful to stage to a specific, named version. For example, when 
iterating on a new doc or design change, you may want to stage to an easily-to-remember
version name, like `new-tooling-section`. 

To push to a named App Engine version, specify the version on the command line:
```

File: docs/README.md [hunk 2]
```diff
@@ -54,3 +54,34 @@ API docs unless you're making changes to the doc comments in the lit-element sou
...
+
+```bash
+npm run deploy -- --version=<VERSION>
+```
+
+This command stages the docs at `https://<VERSION>-dot-polymer-lit-element.appspot.com`.
+
```
Comment (arthurevans) [inline docs/README.md]:
```suggestion

By specifying the version on the command line,  you can update an existing staged version.

```

**Pull request merged**